config/manager/manager.yaml

```diff
@@ -58,11 +58,7 @@ spec:
               cpu: 100m
               memory: 20Mi
           env:
-            - name: RELATED_IMAGE_GRAFANA
-              value: "docker.io/grafana/grafana:9.1.6"
-            - name: WATCH_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.annotations['olm.targetNamespaces']
```

Comment:
I don't think we can remove this config, else the disconnected env will probably get issues.

Reply:
Fixed, the yq command had an issue, not sure how it ever worked :sweat_smile: